test_app/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ endif()
 
 # 3. Add here if the component is compatible with IDF >= v5.0
 if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_GREATER_EQUAL "5.0")
-    list(APPEND EXTRA_COMPONENT_DIRS ../bdc_motor ../led_strip ../sh2lib ../nghttp ../esp_serial_slave_link)
+    list(APPEND EXTRA_COMPONENT_DIRS ../bdc_motor ../led_strip ../sh2lib ../nghttp ../esp_serial_slave_link ../fmt)
```

Comment:
You define compatibility with idf >= 4.1 in `idf_component.yml` but it is compiled only for IDFv5.0.

Could you move this change to line 8? This way it will be compiled for all IDf versions that we have in CI (4.3, 4.4, 5.0, master)

Reply:
Fixed.

Reply:
I'm sorry, but where exactly is it fixed? I can see the `fmt` component added only to build for IDF version >= 5.0

Reply:
Sorry, I tought I  had pushed it yesterday. The modification is available now.